ex5.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def add_vector_to_each_row(matrix, vector):
+    """
+    Function adds a vector to each row of the matrix.
+    :param matrix:(np array) Given matrix.
+    :param vector:(list) Given vector to add to each row in the matrix.
+    :return: New Matrix: the new matrix after adding the vector to each row of the matrix.
+    """
+    result = np.empty_like(matrix)
+    for i in range(len(matrix)):
+        result[i, :] = matrix[i, :] + vector
```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html